fonts/Makefile.am

```diff
@@ -97,6 +105,7 @@ dist_fontdoc_DATA = \
 	Parisienne-OFL.txt \
 	PathwayGothicOne-OFL.txt \
 	Podkova-OFL.txt \
+	PT-OFL.txt \
```

Comment:
nitpick: Call this `PT-LICENSE.txt` since it's a different license from the OFL (Open Font License).